packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -282,6 +282,8 @@ class PointerEventConverter {
               // we don't want to return anything or miss if someone adds a new
               // enumeration to PointerSignalKind.
               throw StateError('Unreachable');
+            default: // ignore: no_default_cases, to allow adding a new [PointerSignalKind] - PointerStylusAuxiliaryAction
```

Comment:
I believe you need a `return null;` here

Reply:
I basically copied this verbatim from this old PR! https://github.com/flutter/flutter/pull/106906/files

Reply:
that PR combines the case with the unknown case.

Reply:
Interesting, okay Ill ad it just in case